modules/transport.js

```diff
@@ -501,9 +501,7 @@ Transport.prototype.onNewBlock = function (block, broadcast) {
 		var broadhash = modules.system.getBroadhash();
 
 		modules.system.update(function () {
-			if (!__private.broadcaster.maxRelays(block)) {
-				__private.broadcaster.broadcast({limit: constants.maxPeers, broadhash: broadhash}, {api: '/blocks', data: {block: block}, method: 'POST', immediate: true});
-			}
+			self.broadcastBlock({limit: constants.maxPeers, broadhash: broadhash}, block);
```

Comment:
`limit` can be skipped, `broadhash` probably also (to research)

Reply:
Yes, the only usage I see was in `getPeers` function where we agreed it is not needed to set.